samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -33,6 +33,24 @@
 
 static const double meanDivisor = 100;
 
+static CGImageRef _CGImageCreate32bppCopy(CGImageRef image) {
```

Comment:
Synced up offline.  The issue is that the original code assumed UIImage implementation details, this change looks at the actual format instead of making assumptions.  

---
In reply to: [98587425](https://github.com/Microsoft/WinObjC/pull/1879#discussion_r98587425) [](ancestors = 98587425)